### ISSUE TITLE
Custom / Native title bar switcher

### ISSUE THREAD

**Issue by chrismou** [radiant-player/radiant-player-electron#7]
Option to switch between custom titlebar (ie, draggable via the Google Music titlebar) or native OS titlebar fallback.

This was a functional requirement in Radiant V1, but on V2 (after seeing the difference of opinion on other projects) I'm thinking this might just be a nice to have to cater for everyone's personal preference.

Depends on #3 for somewhere to put the options

**Comment by jacobwgillespie** [comment]
Yep, once we have a settings layer we can implement this easily.  We would need to change a flag on the window creation call as well as conditionally inject CSS into GPM (so that we don't move the hamburger icon if the custom menu is disabled).
